app/frontend/src/theme.ts

```diff
@@ -37,11 +37,9 @@ const themeOptions: ThemeOptions = {
         },
       },
     },
-    MuiListItem: { gutters: { paddingLeft: 0, paddingRight: 0 } },
-    MuiTabPanel: {
+    MuiFormLabel: {
       root: {
-        padding: 0,
-        paddingTop: spacing(1),
+        "&$focused": { color: "inherit" },
```

Comment:
What does "$" do here?

Reply:
Some MUI magic. This was to fix the colour change on focus. I tried `MuiFormLabel: { focused: { color: "inherit" } }` and I got a helpful warning telling me to do what is here now